controller/lib/core/sensors.h

```diff
@@ -26,44 +26,74 @@ Arduino Nano and the MPXV5004GP and MPXV7002DP pressure sensors.
 #include "network_protocol.pb.h"
 #include "units.h"
 
-// A namespace class for constants related to pressure sensors.
-class PressureSensors {
+class TVIntegrator {
 public:
-  PressureSensors() = delete;
+  void AddFlow(Time now, VolumetricFlow flow);
+  Volume GetTV() const { return volume_; }
+
+private:
+  Time last_flow_measurement_time_ = millisSinceStartup(0);
+  VolumetricFlow last_flow_ = cubic_m_per_sec(0);
+  Volume volume_ = ml(0);
+  bool initialized_ = false;
+};
+
+// Provides calibrated sensor readings, including tidal volume (TV)
+// integrated from flow.
+class Sensors {
+public:
+  Sensors() {}
+
+  // Calibrates the sensors. Must call this before calling any other methods.
+  void Init();
+
+  // Gets gets the current pressure at the patient.
+  Pressure GetPatientPressure();
+  // Gets the volumetric inflow (i.e. rate of air flowing into the patient).
+  VolumetricFlow GetVolumetricInflow();
+  // Gets the volumetric outflow (i.e. rate of air flowing out of the patient).
+  VolumetricFlow GetVolumetricOutflow();
+
+  // get the sensor readings (patient pressure, volumetric flow and tidal
+  // volume) from the sensors
+  SensorReadings GetSensorReadings();
```

Comment:
Can you leave a TODO (from the last PR which touched this) to get rid of the three GetX methods above and just use GetSensorReadings()?

Production code always goes through GetSensorReadings, so that's what I want to test.